k8-util/helm/fluvio-sys/templates/crd_partition.yaml

```diff
@@ -143,6 +143,11 @@ spec:
         format: int64
         description: Leader High Watermark
         jsonPath: .status.leader.hw
+      - name: Base Offset
+        type: integer
+        format: int64
+        description: Base Offset
+        jsonPath: .status.baseOffset
       - name: EOF
```

Comment:
need to bump up helm chart version whenever there is change

Reply:
This value right?
https://github.com/infinyon/fluvio/blob/a5920e3d66fa5c8e6cbf4028542a3069504fdf70/k8-util/helm/fluvio-sys/templates/crd_partition.yaml#L12-L13

Reply:
Here https://github.com/infinyon/fluvio/blob/a5920e3d66fa5c8e6cbf4028542a3069504fdf70/k8-util/helm/fluvio-sys/Chart.yaml#L5